generators/app/templates/src/projectName.jsx

```diff
@@ -16,33 +26,29 @@ class <%= projectClassName %> extends React.Component {
   }
 
   render() {
+    const { name, variant, ...customProps } = this.props;
```

Comment:
where is customProps defined? In the slide panel and arrange its listed explicitly I think.
https://github.com/cerner/terra-slide-panel/pull/5/files#diff-51b83301bcd1b19ca6f080a8074c7fcdR32
https://github.com/cerner/terra-arrange/pull/2/files#diff-20a7f76780d045b34b31540e4cbcba54R30